app/Features/CategoryBanner.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace BlazeWooless\Features;
+
+use BlazeWooless\TypesenseClient;
+
+class CategoryBanner
+{
+    private static $instance = null;
+
+    public static function get_instance()
+    {
+        if (self::$instance === null) {
+            self::$instance = new self();
+        }
+ 
+        return self::$instance;
+    }
+
+    public function __construct()
+    {
+        add_action('product_cat_edit_form_fields', array($this, 'register_category_fields'), 90, 1);
+        add_action('edited_product_cat', array($this, 'save_custom_category_options'), 10, 1);
+
+        add_filter('blaze_commerce_taxonomy_meta_data', array($this, 'register_taxonomy_meta_datas_to_typesense'), 10, 2);
+    }
+
+    public function register_category_fields($tag) {
+        $blaze_commerce_category_show_thumbnail = get_term_meta($tag->term_id, 'blaze_commerce_category_show_thumbnail', true);
+        $blaze_commerce_category_blurred_thumbnail = get_term_meta($tag->term_id, 'blaze_commerce_category_blurred_thumbnail', true);
```

Comment:
Lets remove this code for now, because its not being used at the moment and its fetching data from the database.